content/docs/tasks/security/role-based-access-control/index.md

```diff
@@ -322,6 +322,179 @@ the "black" and "red" ratings in the "Book Reviews" section.
 
 > There may be some delays due to caching and other propagation overhead.
 
+## RBAC policy permissive mode
```

Comment:
IMO, this should be the first entry. We should narrate the tutorial in the way that is best for user to enable the feature (RBAC) in production, rather than describe as separated options.

Reply:
moved to top.